app/routes/start.js

```diff
@@ -25,7 +25,7 @@ const copyprovider = (key) => {
     18: "Problems using the online service",
     19: "If you have any problems using the online service, contact the RPA.",
     20: "Telephone",
-    21: "Telephone: 0300 0200 301",
+    21: "Telephone: 03000 200 301",
```

Comment:
Does this need to be updated?

Reply:
Well, they did not raise bug for this but I received an email and realised that liam has been updated that part. 
here is the link https://eaflood.atlassian.net/browse/GRANTS-1816

Reply:
Well, they did not raise bug for this but I received an email and realised that liam has been updated that part. 
here is the link https://eaflood.atlassian.net/browse/GRANTS-1816

Reply:
I have just spoken to Liam now he said he did it accidentally so I will revert my changs.